src/styles/_atomic.scss

```diff
@@ -0,0 +1,5 @@
+// colors
+
+.clr-secondary {
+    color: $c-secondary;
+}
```

Comment:
it's good practice to insert an empty line at the end of file

Reply:
agree